package.json

```diff
@@ -49,13 +49,14 @@
     "eslint-loader": "^1.3.0",
     "eslint-plugin-react": "^4.2.3",
     "react": "^16.2.0",
-    "react-dom": "^16.2.0",
     "react-addons-test-utils": "^0.14.8",
+    "react-dom": "^16.2.0",
     "webpack": "3.7.1",
+    "webpack-bundle-analyzer": "^2.9.2",
     "webpack-dev-server": "2.9.1"
   },
   "dependencies": {
-    "prop-types": "15.6.0",
-    "mediaquery": "0.0.4"
+    "prop-types": "^15.0.0",
```

Comment:
This is to make it easier for `prop-types` to be deduped. This library doesn't seem to be depending on the specific version of `prop-types`.